sdk/mobile-center/src/test/java/com/microsoft/azure/mobile/AbstractMobileCenterServiceTest.java

```diff
@@ -18,7 +18,10 @@
 import static com.microsoft.azure.mobile.utils.PrefStorageConstants.KEY_ENABLED;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.any;
```

Comment:
Not for this line. Can you static import `Assert.assertSame` in the file? We have two usages. Feel free to do it for `assertEquals` and `assertNull` (one occurrence in the file).